spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/SqlRowWriter.scala

```diff
@@ -23,9 +26,16 @@ class SqlRowWriter(val table: TableDef, val selectedColumns: IndexedSeq[ColumnRe
     require(row.size == columnNames.size, s"Invalid row size: ${row.size} instead of ${columnNames.size}.")
     for (i <- 0 until row.size) {
       val colValue = row(i)
-      buffer(i) = converters(i).convert(colValue)
+      buffer(i) = Try(converters(i).convert(colValue)) match {
+        case Success(value) => value
+        case Failure(ex: IllegalArgumentException) =>
+          val columnType = columnTypes(i).scalaTypeName
+          throw new TypeConversionException(
+            s"Cannot convert value $colValue of column ${columnNames(i)} to type $columnType", ex)
```

Comment:
The message will look like `Cannot convert value somevalue of column somecolumn to type Int` without any wrapping of `somevalue` (like single or double quotes). Any suggestions on how to improve readability?